packages/api/src/command/mapping/patient.ts

```diff
@@ -31,6 +31,13 @@ export async function getPatientMapping({
   return existing;
 }
 
+export async function getPatientMappingsForPatient(where: {
+  patientId: string;
+  source?: string;
```

Comment:
Yea, was just thinking about this. Might want to add cxId to this table as right now, we only sure correct access because we call GetPatientOrFail along the returned patientId, which requires the cxId. But, might want to check this access either table level (i.e. make cxId a column) or in the command check whether the cxId has access to the patient.

Reply:
Done -- added cxId column that's used in where conditions to ensure access at the command level.